oat-sa/extension-tao-testtaker
# fix: update and add translation for de-DE

Related to:
https://oat-sa.atlassian.net/browse/AUT-2349
https://github.com/oat-sa/tao-core/pull/3681

**PR CONTAINS:**

- Addition of missing translations for extension for `de-DE` required by ticket (TestTaker tab)

STEPS TO TEST:

Go to the `testTakers` tab in German and check that '`Last updated on`' is translated all elements are translated :

CURRENT RESULT:
Is not translated

EXPECTED RESULT:

Element shoudl be translated as bellow:
![image](https://user-images.githubusercontent.com/60346520/203811828-49243d34-cf17-4128-8a7f-80d3b2dfa09d.png)
